tests/test_triples_factory.py

```diff
@@ -86,6 +87,9 @@ def test_automatic_inverse_detection(self):
 
     def test_automatic_incomplete_inverse_detection(self):
         """Test if the TriplesFactory detects that the triples contain incomplete inverses and creates correct ids."""
+        # comment(mberr): from my pov this behaviour is faulty: the triples factory is expected to say it contains
+        # inverse relations, although the triples contained in it are not the same we would have when removing the
+        # first triple, and passing create_inverse_triples=True.
```

Comment:
@cthoyt @lvermue 
The behaviour was changed in this PR. If you approve these changes, I'll adapt the unittest.

Reply:
You're probably right - using the `_inverse` suffix to assume that there were already inverses was a bit sketchy. I'm okay with changing this behavior - thanks for flagging it for specific discussion

Reply:
Right now, I still run the check. If I find something looking like an inverse, I emit a warning, and discard all triples with these relations. They get re-created automatically when creating the ID-based training instances.

Reply:
I think that's a really good compromise

Reply:
I agree however that this still is sketchy since it relies on a very specific naming scheme. For content-based inverse relation detection we already have the sealant. Running its detection whenever we create a triples factory might however slow things down.

We could add a flag to manually select to trigger a check, but then again, the user could just directly use the sealant.

Reply:
Shouldn't we delete it completely then?

Reply:
Let's take care of that in the future PR where we make the functionality work

Reply:
I am also in favour of deleting it.

Reply:
Okay I didn’t have a strong opinion, so please @mberr delete the deprecated code 💯